libraries/botframework-expressions/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "botframework-expressions",
+  "description": "Common Expression Language",
+  "version": "4.1.6",
+  "license": "MIT",
+  "keywords": [
+    "botbuilder",
+    "botframework",
+    "expression"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/botbuilder-js.git"
+  },
+  "main": "./lib/index.js",
+  "typings": "./lib/index.d.ts",
+  "dependencies": {
+    "@types/lru-cache": "^5.1.0",
+    "@microsoft/recognizers-text-data-types-timex-expression": "^1.1.4",
+    "antlr4ts": "0.5.0-alpha.1",
+    "jspath": "^0.4.0",
+    "lodash": "^4.17.15",
+    "lru-cache": "^5.1.1",
+    "moment": "2.24.0",
+    "moment-timezone": "^0.5.25",
+    "xml2js": "^0.4.19",
+    "xmldom": "^0.1.27",
+    "xpath": "0.0.27"
```

Comment:
same as other comment. we should ensure these will work in a browser.

Reply:
remove xml and xpath builtin function